docs/admin/deployment.rst

```diff
@@ -72,6 +72,18 @@ you create:
    Set to 'gitlab' if you want to use 'GitLab' (corresponding GITLAB_* settings must be set if required).
    Set to 'google' if you want to use 'Google' (corresponding GOOGLE_* settings must be set).
 
+``BADGES_PROMOTION_THRESHOLDS``
+   Optional. A comma-separated list of numeric thresholds for different levels of the 
+   Community Builder badge.
+
+``BADGES_START_DATE``
+   Optional. Specifies the start date from which user activities count towards badge achievements. 
+   This variable should be in YYYY-MM-DD format.
+
+``BADGES_TRANSLATION_THRESHOLDS``
```

Comment:
Question for a potential follow-up: I'm not sure we should have one setting shared between two different badges.
The numbers happen to match in our proposal, but now we're enforcing that for anyone wanting to use badges.

Reply:
Somehow missed this comment

> I'm also thinking that we should probably have separate lists for Review Master and Translation Champion lists, but that's a separate issue.